webp_backport/src/androidTest/java/de/marcreichelt/webp_backport/FileSignatureCheckerTest.java

```diff
@@ -0,0 +1,90 @@
+package de.marcreichelt.webp_backport;
+
+import android.test.AndroidTestCase;
+
+import org.apache.commons.io.IOUtils;
+
+import java.io.InputStream;
+
+import de.marcreichelt.webp_backport.test.R;
+
+public class FileSignatureCheckerTest extends AndroidTestCase {
+
+    static {
+        WebPBackport.loadLibrary();
+    }
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+        assertTrue("native WebP library must be loaded for tests to be able to run", WebPBackport.librarySuccessfullyLoaded);
+    }
+
+    public void testByteArrayContains() throws Exception {
+        assertTrue(FileSignatureChecker.byteArrayContainsAt(new byte[]{1, 2, 3, 4}, new byte[]{1, 2, 3, 4}, 0));
```

Comment:
new byte[]{1, 2, 3, 4} is used a lot, deserves a constant ARRAY_1234 or equivalent.